doc/containers.md

```diff
@@ -67,6 +67,8 @@ Other hashmap function:
 This is a traditional Lisp conscell data type, pair of values (car, cdr).  It can
 be used to create a linked list of values.
 
+Note: an empty list is not nil and nil is not an empty list.
```

Comment:
this is true now right?

Reply:
No, that is something I think maybe we should do.  Has not been implemented yet so if you have an opinion let me know.

Reply:
It's on the radar because of the tests for sure.